daemon/src/projection.rs

```diff
@@ -58,6 +59,10 @@ pub struct Update<T>(pub T);
 #[derive(Clone, Copy)]
 pub struct CfdChanged(pub OrderId);
 
+/// Indicates that all Cfd have to be loaded initially
```

Comment:
I found this comment more confusing than helpful, we might as well delete it or say something like:

"Perform the bulk initialisation of the CFD feed"